CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 [0.1.0] - 2022-XX-XX
 --------------------
 * Updated the GitHub action version numbers
+* Maintenance
```

Comment:
If you're going to introduce a Maintenance section, be sure to add previous maintenance updates under it.

Reply:
updated